ios/Classes/ComAppceleratorUrlSessionSessionProxy.m

```diff
@@ -25,6 +25,15 @@ - (id)_initWithPageContext:(id<TiEvaluator>)context andArguments:(NSDictionary*)
     
     return self;
 }
+-(void)_initWithProperties:(NSDictionary *)properties
+{
+    sessionConfiguration = [properties objectForKey:@"configuration"];
+    [self rememberProxy:sessionConfiguration];
+    session = [NSURLSession sessionWithConfiguration:[sessionConfiguration sessionConfiguration]
+                                            delegate:(id<NSURLSessionDelegate>)[[UIApplication sharedApplication] delegate]
+                                       delegateQueue:nil];
+    [super _initWithProperties:properties];
```

Comment:
If I use tirulesession.createURLSession(opts) (I call this the legacy method) , it's allright. But if I create the proxy directly by doing tirulesession.createSession(opts), then it doesnt call `_initWithPageContext:`